about.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sorting-Algorithms-in-Javascript</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/77ebb379f6.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="../css/main.css">
+</head>
```

Comment:
Brak indentacji (wcięcia) + niepotrzebne spacje dookoła elementu `<head>`

Reply:
Tutaj niestety muszę zwalić winę na edytor, przykładowo usunąłem te spacje ale kiedy wciskam ctrl+s aby kod się sformatował to formatuje mi właśnie w ten sposób.